### DISCUSSION TITLE
Audio Visualizer Overlay

### DISCUSSION THREAD

**Discussion by ShashTheEpik** [rocksdanister/lively#1379] [Ideas]
There should be a tab in the lively app called "Overlays". These would function by having a library of user and lively-based wallpapers that overlay on the current wallpaper. The Audio Visualizer should be in this category where its exact position, size and colour can be changed. Sorry if I sound like I'm demanding something I just want to suggest a feature.

**Comment by rocksdanister** [comment]
I usually stay away from this because [Rainmeter](https://www.rainmeter.net/) does a good job already with widgets.

**Reply by ShashTheEpik** [reply]
ok
